src/v2/guide/accessibility.md

```diff
@@ -0,0 +1,224 @@
+
+The World Health Organization estimate that 15% of the world's population have some form of disability, 2-4% of them severely so.
+
+In addition to helping users with disabilities, making a website accessible will help the rest of your users, too. Adding subtitles to a video will help both your deaf and hard-of-hearing users and your users who are in a loud environment and can't hear their phone. Similarly, making sure your text isn't too low contrast will help both your low-vision users and your users who are trying to use their phone in bright sunlight.
+
+The W3C have published a list of guidelines for making websites accessible, called the [Web Content Accessibility Guidelines (WCAG)][wcag 2.0]. It's worth a read, but you might find it a bit dry, and it's out of date - it was written before Single Page Applications and libraries like Vue existed so there are no specific guidelines for that.
+
+There are a huge range of disabilities, which can be divided roughly into four categories:
+
+- *Visual impairments*, such as being blind, having low vision, or colourblindness. These people may make text bigger, increase the contrast of their screen, or use a screen reader or braille display.
+- *Motor impairments*—a huge category of impairments—can include people who browse using only the keyboard, use voice recognition software, or use [switch technology].
+- *Hearing impairments*, from being partially deaf to profoundly deaf.
+- *Cognitive impairments*, such as dyslexia, epilepsy, intellectual disabilities, and ADHD (very much not a complete list).
+
+There's a lot of technology that people with disabilities utilise to use their computers, but often that technology isn't useful unless we take measures to ensure it works. For example, low vision users, blind users and users with dyslexia may use screen readers to read out what's on the screen—but if there are images on the page the screen reader won't know what to read out unless we tell it using alt text. This guide will outline a number of common problems people with disabilities have when browsing websites and explain what we can do to help.
+
+## Basic accessibility
+
+Accessibility in standard websites, such as putting alt text on images and making sure your text is large enough to read, has been covered many times before, so instead of rewriting what has already been said here, here's a couple articles covering the topic:
+
+@todo: add some links
+
+## Accessibility in JavaScript-rich websites
+
+JavaScript allows us to make powerful, interactive websites and applications, with features from animations to client-side routing instead of having to hit the server every time the user goes to a new page. Unfortunately, some accessibility technology, especially screen readers, doesn't work so well with these websites. 
```

Comment:
"JavaScript allows us to make powerful, interactive websites and applications, with features from animations to client-side routing instead of having to hit the server every time the user goes to a new page. Unfortunately, some accessibility technology, especially screen readers, doesn't work so well with these websites. "

maybe something like

"JavaScript allows for powerful, interactive websites and applications. Its usage can be simple or complex. Unfortunately, some assistive technologies struggle to support this environment. With this in mind what can we do to accommodate  these tools added needs?"

Reply:
Added a comma to break up "With this in mind, what can we..."